packages/repository-json-schema/src/build-schema.ts

```diff
@@ -108,15 +108,38 @@ export function metaToJsonProperty(meta: PropertyDefinition): JSONSchema {
   const wrappedType = stringTypeToWrapper(propertyType);
   const resolvedType = resolveType(wrappedType);
 
+  // create a shallow copy of meta, so we can delete what we don't need
+  const metaCopy = {...meta};
+  // delete everything we don't need
+  [
```

Comment:
How do you decide which properties are not needed?

Reply:
1. It is defined in the `PropertyDefinition`, but not in the AJV's keywords specs.
2. `type` & `description`, since they will be written by the original code.